composer.json

```diff
@@ -49,7 +53,7 @@
     "enable-patching": true,
 	"patches": {
 		"drupal/[module_name]": {
-			"Note regarding the nature of the patch being applied": "https://www.drupal.org/files/issues/[patch_name].patch"     
+			"Note regarding the nature of the patch being applied": "https://www.drupal.org/files/issues/[patch_name].patch"
 		}
 	}
```

Comment:
"patches" section also had pre-existing extra indentation, would be good to fix that up while you're in here.

Reply:
All Set! I checked and fixed indentation for the whole file. Thanks.